packages/web-client/app/components/card-pay/create-merchant-workflow/prepaid-card-choice/index.hbs

```diff
@@ -27,7 +28,29 @@
       />
     </CardPay::LabeledValue>
     <CardPay::LabeledValue @label="Pay Using Prepaid Card" class="prepaid-card-choice__field-label">
-      {{!-- TODO: card picker --}}
+      {{#if this.selectedPrepaidCard}}
+        <div class="prepaid-card-choice__change-card">
+          <CardPay::CardPicker::CardOption
+            @card={{this.selectedPrepaidCard}}
+            data-test-prepaid-card-choice-selected-card
+          />
+          {{#if (and (not @isComplete) (not-eq this.creationState "in-progress"))}}
+            <CardPay::CardPicker
+              @cards={{this.prepaidCards}}
+              @chooseCard={{this.choosePrepaidCard}}
+              @selectedCard={{this.selectedPrepaidCard}}
+              @selectedCardAddress={{this.selectedPrepaidCard.address}}
```

Comment:
Just looking at this invocation and how the component is defined, it looks like `selectedCardAddress` could just be removed and accessed in the component as `@selectedCard.address`? 🤔